src/runtime/pkg/govmm/qemu/qemu.go

```diff
@@ -283,6 +286,9 @@ type Object struct {
 
 	// Prealloc enables memory preallocation
 	Prealloc bool
+
+	// Policy with which guest VM is initialized. Relevant for sev-guest and sev-snp-guest objects.
+	Policy uint64
```

Comment:
The weird thing is that the meaning of the policy is totally different in SEV(-ES) and in SNP. It should have different default values (when the user doesn't specify it), etc. Not sure if that warrants holding two fields (SevPolicy, SnpPolicy), but it's something to keep in mind.

Reply:
That is true... for me personally I don't mind this dual use of the field, though I might be too much in the thick of it. It might also make sense to define default policy in Kata-Containers as well and not fall back to QEMU's default as is the case with SEV, though this is not something to be addressed in this PR...

Reply:
I'm not familiar with SEV(-ES) / SNP, so I'd like to ask an input from someone from their side to decide what's the best to do here.